DigitalLearningSolutions.Web/Controllers/ResetPasswordController.cs

```diff
@@ -0,0 +1,39 @@
+namespace DigitalLearningSolutions.Web.Controllers
+{
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Services;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class ResetPasswordController : Controller
+    {
+        private readonly IPasswordResetService passwordResetService;

```

Comment:
Super NIP, but could we add an empty line before the constructor for better readability (I think the autoformatter should be doing it by default).